packages/gatsby-plugin-clerk/src/ssr/types.ts

```diff
@@ -6,9 +6,9 @@ export type WithServerAuthResult<CallbackReturn> = (props: GetServerDataProps) =
 
 export type GetServerDataPropsWithAuth<Options extends WithServerAuthOptions = any> = GetServerDataProps & {
   auth: ServerSideAuth;
-} & (Options extends { loadSession: true } ? { session: Session | null } : {}) &
-  (Options extends { loadUser: true } ? { user: User | null } : {}) &
-  (Options extends { loadOrg: true } ? { organization: Organization | null } : {});
+} & (Options extends { loadSession: true } ? { session: Session | null } : object) &
```

Comment:
Just curious - which rule prevents `object` from throwing a warning? I *think* that eslint used to complain about direct usage of `object` but I might be wrong :)

Reply:
```
Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `object` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead.
- If you really want a type meaning "any non-nullish value", you probably want `NonNullable<unknown>` instead.eslint[@typescript-eslint/ban-types](https://typescript-eslint.io/rules/ban-types)
```

So that's why I chose `object` for now. I'd prefer `Record<string, unknown>` but that's a convention choice we can enforce through ESLint, too, once we decided

Reply:
Great - just checked locally and everything passes so I was probably wrong about `object` causing an eslint warning,